### ISSUE TITLE
The link in README for installation instructions leads to a dead end

### ISSUE THREAD

**Issue by Ash110** [bridgedxyz/assistant#6]
**Describe the bug**
The link in the README to compile locally, i.e, https://github.com/bridgedxyz/assistant/blob/master/plugin leads nowhere. It's a missing page

**Expected behavior**
Should lead to an updated page with clear instructions on how to build and use locally till Figma approves plugin

**Comment by softmarshmallow** [comment]
duplicate of #2 
plugin is being reviewed by figma team, until then, you have to run locally.

**Comment by Ash110** [comment]
No, not the Figma link. I'm talking about the link on the README which is supposed to show how to build locally
_link is temporarily dead. it's being reviewed by figma team. until this plugin is available publicly, you have to build it on local and run it. guideline at here_

**Comment by softmarshmallow** [comment]
Oh, I see, i'll update it. thanks for pointing out.

**Comment by softmarshmallow** [comment]
https://github.com/bridgedxyz/assistant/commit/4cbee4886a59eafc9c09eaf20537c093e4fff953
